network/p2p/acp118/handler_test.go

```diff
@@ -26,25 +26,23 @@ func TestHandler(t *testing.T) {
 	tests := []struct {
 		name         string
 		cacher       cache.Cacher[ids.ID, []byte]
-		verifier     Verifier
```

Comment:
Don't think we need this diff? The `testVerifier` type now takes in a slice of errors when it's created